keras/layers/convolutional.py

```diff
@@ -377,7 +377,7 @@ def get_config(self):
         return dict(list(base_config.items()) + list(config.items()))
 
 
-class AtrousConv2D(Convolution2D):
+class AtrousConvolution2D(Convolution2D):
```

Comment:
In that case this name should also be modified in all calls to it with the class definition (super).